bin/mail/2016-09-12-back-to-school/generate_list_functions.rb

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+require 'active_support'
+require 'active_support/core_ext/object/blank'
+
+def query_dashboard_teachers(query)
+  {}.tap do |results|
+    DASHBOARD_DB.fetch(query).each do |teacher|
+      email = teacher[:email]
+      results[email] = {email: email, name: teacher[:name], id: teacher[:id]}
+    end
+  end
+end
+
+def query_workshop_section_ids
+  DASHBOARD_DB.fetch("SELECT id FROM sections WHERE section_type LIKE '%workshop'").map(:id)
```

Comment:
Yes, it's sequel's version of ActiveRecord's `pluck` to return an array of column values. See http://sequel.jeremyevans.net/rdoc/files/doc/querying_rdoc.html#label-As+an+Array+of+Column+Values